lib/rack/utils.rb

```diff
@@ -551,6 +551,7 @@ def names
       428 => 'Precondition Required',
       429 => 'Too Many Requests',
       431 => 'Request Header Fields Too Large',
+      451 => 'Unavailable For Legal Reasons',
```

Comment:
@jeremy Ha! Was just writing the comment when I noticed that the [IANA CSV](https://www.iana.org/assignments/http-status-codes/http-status-codes-1.csv) *was updated* to include the new status code. So I updated per the comment on line 503. :tada: